src/components/ToDo/ToDo.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Card, Badge, Col } from 'react-bootstrap';
+import { TypeToDo } from '../../types';
+
+export const ToDo = ({ todo: { title, userName, completed } }) => (
```

Comment:
```suggestion
export const Todo = ({ todo: { title, userName, completed } }) => (
```

Reply:
I've renamed files and components. Locally, everything works, but it won't be staged for a new commit because of the new name.  I really don't know how to commit and push this file
![Annotation 2021-01-25 205653](https://user-images.githubusercontent.com/22053528/105752521-13414880-5f50-11eb-9be9-98ed244d6da2.png)